src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static int getRandomBall() {
+        Random random = new Random();
+        int value = random.nextInt(100);
+        return value;
+    }

```

Comment:
It should not be static and should return a `Ball` instance with color and number fields initialized, not just a number. And stick to the other comment about the new random in the method